modules/os2forms_permissions_by_term/README.md

```diff
@@ -0,0 +1,25 @@
+# OS2Forms permission by term module
+This module implements permission by term access restrictions
+on several lists and entity displays related to webform and maestro.
+
+## Setup configuration
+Add to your settings.php or local.settings.php
+```
+$config['permissions_by_term.settings'] = [
```

Comment:
Ẁhy this change need to be done manually?
Can't this bedone from module install_hook?

Reply:
Adding it to the readme like this has two main purposes.
1.) Documentation
This module supports a specific configuration of the permissions by term module. If it was just added as a config change in the install hook, it could easily be changed by a site administrator or others which would cause unexpected results. If the Readme is followed and the lines added to settings.php the config will always match the needed behavior of the permissions_by_term module. It also helps to document some needed configuration changes without attempting to explain the config settings of another module.

2.) Ownership
The config of the permissions_by_term module is not (in my opinion) for this module to determine. If for some reason you have a setup where permissions by term module needs to have a different configuration, it would seem too invasive for this module to change an existing config. 
The config is either: Owned by the site (In case of an existing site when this module is activated) or by the permissions_by_term module itself (New site creation).

Reply:
Added more directions to the Readme file.